dsc/workflows/base/__init__.py

```diff
@@ -63,37 +69,45 @@ def metadata_mapping(self) -> dict:
     @final
     @classmethod
     def load(
-        cls, workflow_name: str, collection_handle: str, batch_id: str
-    ) -> BaseWorkflow:
+        cls,
+        workflow_name: str,
+        **kwargs: str | tuple | None,
+    ) -> Workflow:
         """Return configured workflow class instance.
 
         Args:
             workflow_name: The label of the workflow. Must match a key from
             config.WORKFLOWS.
-            collection_handle: The handle of the DSpace collection to which the batch will
-            be submitted.
-            batch_id: The S3 prefix for the batch of DSpace submissions.
+            **kwargs: Includes required arguments (collection_handle, batch_id,
+            email_recipients) as well as optional arguments (s3_bucket, output_queue)
+            that override the class's default values.
         """
         workflow_class = cls.get_workflow(workflow_name)
-        return workflow_class(
-            collection_handle=collection_handle,
-            batch_id=batch_id,
-        )
+        filtered_kwargs = {
+            key: value for key, value in kwargs.items() if value is not None
+        }
+        return workflow_class(**filtered_kwargs)  # type: ignore [arg-type]
```

Comment:
Not sure if I'm doing this right given the need for `# type: ignore [arg-type]` so let me know if there's a better way!